src/main/kotlin/org/stellar/walletsdk/Auth.kt

```diff
@@ -43,7 +43,15 @@ class Auth(
       .addQueryParameter("account", accountAddress)
       .addQueryParameter("home_domain", homeDomain)
 
+    if (!memoId.isNullOrBlank()) {
+      authURL.addQueryParameter("memo", memoId)
+    }
+
```

Comment:
Should we validate that the string is a valid ID memo?

Reply:
Yes! Added a check to make sure memo ID is a positive integer. :+1: